CHANGELOG.md

```diff
@@ -6,39 +6,48 @@ CHANGELOG
-- Allow for mounting `home` as an EFS or FSx external shared storage via the `SharedStorage` section of the config file.
 - Add support for Rocky Linux 8, only using a `CustomAmi` created through `build-image` process. No official Rocky8 Linux AMIs will be published.
-- Make `InstanceType` an optional configuration parameter when configuring `CapacityReservationTarget/CapacityReservationId` in the compute resource.
 - Add `Scheduling/ScalingStrategy` parameter to control job-level scaling strategy for node to be resumed by Slurm.
   Possible values are `all-or-nothing`, `greedy-all-or-nothing`, `best-effort`, with `all-or-nothing` being the default.
+- Add the option to use EFS storage instead of NFS exports from the head node root volume for intra-cluster shared ParallelCluster, Intel, Slurm, login node, and `/home` data.
+- Allow for mounting `home` as an EFS or FSx external shared storage via the `SharedStorage` section of the config file.
+- Add new parameter `SlurmSettings/MungeKeySecretArn` to permit to use an external user-defined MUNGE key from AWS Secrets Manager.
+- Add `Monitoring/Alarms/Enabled` parameter to toggle Amazon CloudWatch Alarms for the cluster.
+- Add head node alarms to monitor EC2 health checks, CPU utilization and the overall status of the head node.
+- Add support for Data Repository Associations when using `PERSISTENT_2` as `DeploymentType` for a managed FSx for Lustre.
+- Add `Scheduling/SlurmSettings/Database/DatabaseName` parameter to allow users to specify a custom name for the database on the database server to be used for Slurm accounting.
+- Make `InstanceType` an optional configuration parameter when configuring `CapacityReservationTarget/CapacityReservationId` in the compute resource.
 - Add possibility to specify a prefix for IAM roles and policies created by ParallelCluster API.
 - Add possibility to specify a permissions boundary to be applied for IAM roles and policies created by ParallelCluster API.
 
 **CHANGES**
 - Upgrade Slurm to 23.02.7 (from 23.02.6).
-- Changed cluster alarms naming convention to '[cluster-name]-[component-name]-[metric]'.
-- Add head node alarms to cluster dashboard.
-- Add support for Python 3.10 in aws-parallelcluster-batch-cli.
-- Remove `all_or_nothing_batch` resume configuration parameter, in favor of the new `scaling_strategy` parameter 
+- Upgrade NVIDIA driver to version 535.129.03.
+- Upgrade CUDA Toolkit to version 12.2.2.
+- Use Open Source NVIDIA GPU drivers (OpenRM) as NVIDIA kernel module for Linux instead of NVIDIA closed source module.
+- Remove `all_or_nothing_batch` resume configuration parameter, in favor of the new `scaling_strategy` parameter
   that can be set using `Scheduling/ScalingStrategy` cluster configuration.
-- Change default EBS volume types in ADC regions from gp2 to gp3, for both the root and additional volumes.
+- Add head node alarms to cluster dashboard.
```

Comment:
This is an enhancement. We'd mention the newly introduced alarms.

Reply:
moved